core/src/main/java/com/sequenceiq/cloudbreak/service/image/HdpInfoSearchService.java

```diff
@@ -6,11 +6,11 @@
 
 import javax.inject.Inject;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
why did you switch from org.springframework.util.StringUtils to apache Stringutils?